typescript/infra/config/environments/mainnet/helloworld.ts

```diff
@@ -0,0 +1,18 @@
+import { HelloWorldConfig } from '../../../src/config';
+
+import { MainnetChains, environment } from './chains';
+import helloWorldAddresses from './helloworld/addresses.json';
+
+export const helloWorld: HelloWorldConfig<MainnetChains> = {
+  addresses: helloWorldAddresses,
+  kathy: {
+    docker: {
+      repo: 'gcr.io/abacus-labs-dev/abacus-monorepo',
+      tag: 'sha-f58131a',
```

Comment:
Wondering if this should be somewhere easier to find? But nbd

Reply:
Yeah this definitely isn't ideal. Just because this is the pattern we've gone with for other infra so far, I think it's best to keep as is for now